GOVERNANCE.md

```diff
@@ -0,0 +1,39 @@
+# Porter Project Goveranance
+
+This document outlines how the Porter project governs itself. The Porter project
+consists of all of the repositories in the https://github.com/getporter
+organization. 
+
+Everyone who interacts with the project must abide by our [Code of Conduct]. 
+
+## Legal
+
+The Porter project is in the [CNCF Sandbox](https://www.cncf.io/sandbox-projects/). ⛱
+
+## Roles
+
+We have a [Contribution Ladder] that explains the capabilities and
```

Comment:
Somewhere here should say explicitly that the contributor ladder includes the steps to progress to different roles (in other words, that this is the place to find out how to become a maintainer).

Reply:
Right now this makes it sound like the contribution ladder file just explains what the different roles and responsibilities are.